examples/prisma-react/README.md

```diff
@@ -0,0 +1,6 @@
+```
+npm install
+npx prisma migrate dev --name init
+npx prisma db seed
+npm run dev
+```
```

Comment:
Good readme 👌. (Makes me actually think that the other readmeS are too verbose.)